sources/packages/api/src-sql/EducationProgramsOfferings/Rollback-add-course-load-and-cleanup.sql

```diff
@@ -0,0 +1,19 @@
+--Add column lacks_study_dates to sims.education_programs_offerings.
+ALTER TABLE
+    sims.education_programs_offerings
+ADD
+    COLUMN IF NOT EXISTS lacks_study_dates BOOLEAN NOT NULL;
```

Comment:
we cannot add a NOT NULL column without default assuming data will be present.

Reply:
I assume rollback will fail.

Reply:
Yes, I missed it